ee/api/enterprise_event_definition.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework import serializers
+
+from ee.models.event_definition import EnterpriseEventDefinition
+
+
+class EnterpriseEventDefinitionSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = EnterpriseEventDefinition
+        fields = (
```

Comment:
See comment on `migration/0005_...`, I think it might be easier to use `EnterpriseDefinition` as base and inner join `EnterpriseEventDefinition` where available.